pandas/core/frame.py

```diff
@@ -7146,11 +7148,11 @@ def corr(self, method="pearson", min_periods=1):
             * kendall : Kendall Tau correlation coefficient
             * spearman : Spearman rank correlation
             * callable: callable with input two 1d ndarrays
-                and returning a float. Note that the returned matrix from corr
-                will have 1 along the diagonals and will be symmetric
-                regardless of the callable's behavior.
+              and returning a float. Note that the returned matrix from corr
+              will have 1 along the diagonals and will be symmetric
+              regardless of the callable's behavior.
 
-                .. versionadded:: 0.24.0
+              .. versionadded:: 0.24.0
```

Comment:
did you check if the html version of this docstring looks good? those indentations are always tricky

Reply:
This is correct (it has the same indentation as the list item)